PelotonEppSdk/Models/EventItem.cs

```diff
@@ -78,6 +82,28 @@ internal class event_item
                     CustomFields = customFields
                 };
             }
+
+            public static explicit operator event_item(EventItem ei)
+            {
+                if (ei == null) return null;
+
+                ICollection<event_custom_field> customFields = null;
```

Comment:
would this work instead (haven't tested it):
var customFields = ei?.CustomFields.Select(cf=> (event_custom_field) cf).ToList()

It would remove the need for the explicit declaration and the if statement